frontend/src/js/directives/directives.js

```diff
@@ -1,13 +1,8 @@
 // define directives here
-
 (function() {
-
     'use strict'
-
     // dynamic header directive
-    angular
-        .module('evalai')
-        .directive('dynHeader', dynHeader);
+    angular.module('evalai').directive('dynHeader', dynHeader);
```

Comment:
@itaditya: After applying prettify this should not change. It shows changes due to previous formatting done by you.
Just fix this. this should look like this.
(remember that 4 spces/1 tab for indentation)
````
angular
    .modules('evalai')
        .directive('dynHeader', dynHeader);
````

once you format code like this prettify(`Ctrl + shift + H`) will take care of everything.